app/helpers/audits_helper.rb

```diff
@@ -238,11 +238,31 @@ def main_object?(audit)
     main_objects_names.include?(type)
   end
 
-  def key_to_class(key, audit)
+  def find_auditable_type_class(audit)
     auditable_type = (audit.auditable_type == 'Host::Base') ? 'Host::Managed' : audit.auditable_type
+    auditable_type.constantize
```

Comment:
note to myself: this may be a problem if the class no longer exists but we already had this problem, so ignoring for now